keps/sig-auth/1205-bound-service-account-tokens/README.md

```diff
@@ -265,53 +258,176 @@ service account token on behalf of pods running on that node. The
 NodeRestriction admission controller will require that these tokens are pod
 bound.
 
-### Safe Adoption
+### ServiceAccount Admission Controller Migration
+
+#### Prerequisites
+
+Before migration to a version with `BoundServiceAccountVolume=true`, cluster
+operators should make sure:
+
+1.  Set feature gate `TokenRequest=true`. (default to `true` since 1.12)
+
+    - This feature requires the following flags to the API server:
+      - `--service-account-issuer`
+      - `--service-account-signing-key-file`
+      - `--service-account-key-file`
+      - `--api-audiences` (default to `--service-account-issuer`)
+
+2.  Set feature gate `TokenRequestProjection=true`. (default to `true` since
+    1.12)
+
+3.  Update all workloads to newer version of officially supported Kubernetes
+    client libraries to reload token:
+
+    - Go: >= v11.0.0
```

Comment:
nit: client-go versioning is now of the form v0.X.0, where X matches the kubernetes minor version, so this is >= v11.0.0 or >= v0.15.7

Reply:
done